<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>8</number>
   </property>
   <item>
    <widget class="QFrame" name="options_frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="xmlCategory" stdset="0">
      <string>file</string>
     </property>
     <layout class="QFormLayout" name="options_formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="shot_label">
        <property name="text">
         <string>Shot:</string>
        </property>
        <property name="buddy">
         <cstring>shot_lineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="shot_frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="shot_horizontalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="shot_lineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="shot_toolButton">
           <property name="text">
            <string>Change...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="discipline_label">
        <property name="text">
         <string>Discipline:</string>
        </property>
        <property name="buddy">
         <cstring>discipline_comboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="discipline_comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="xmlTag" stdset="0">
         <string>discipline</string>
        </property>
        <item>
         <property name="text">
          <string>[any]</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>matchmove</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>model</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>lookdev</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>texturing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>rigging</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>animation</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>fx</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>groom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>lighting</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>roto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>prep</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>comp</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>precomp</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="description_label">
        <property name="text">
         <string>Description:</string>
        </property>
        <property name="buddy">
         <cstring>description_comboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="description_comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAtTop</enum>
        </property>
        <property name="xmlTag" stdset="0">
         <string>description</string>
        </property>
        <property name="storeHistory" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="version_label">
        <property name="text">
         <string>Version:</string>
        </property>
        <property name="buddy">
         <cstring>version_spinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QFrame" name="version_frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="version_horizontalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="version_spinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="versionPadding_label">
           <property name="text">
            <string>Padding:</string>
           </property>
           <property name="buddy">
            <cstring>versionPadding_spinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="versionPadding_spinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>4</number>
           </property>
           <property name="xmlTag" stdset="0">
            <string>padding</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="version_horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="nativeDialog_toolButton">
           <property name="toolTip">
            <string>Open the application's native dialog for saving files</string>
           </property>
           <property name="text">
            <string>Native Dialog...</string>
           </property>
           <property name="iconSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="noBackground" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="output_frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QFormLayout" name="output_formLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="filename_label">
        <property name="text">
         <string>File name:</string>
        </property>
        <property name="buddy">
         <cstring>filename_lineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="filename_frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="filename_horizontalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="filename_lineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>shot_lineEdit</tabstop>
  <tabstop>shot_toolButton</tabstop>
  <tabstop>discipline_comboBox</tabstop>
  <tabstop>description_comboBox</tabstop>
  <tabstop>version_spinBox</tabstop>
  <tabstop>filename_lineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
